lib/matplotlib/axes/_axes.py

```diff
@@ -3723,9 +3723,10 @@ def boxplot(self, x, notch=None, sym=None, vert=None, whis=None,
         Draw a box and whisker plot.
 
         The box extends from the first quartile (Q1) to the third
-        quartile (Q3) of the data, with a line at the median.  The
-        whiskers extend from the box by 1.5x the inter-quartile range
-        (IQR).  Flier points are those past the end of the whiskers.
+        quartile (Q3) of the data, with a line at the median. The
+        whiskers extend from the box to the farthest point within
+        1.5x the inter-quartile range (IQR) distance. Flier points
+        are those past the end of the whiskers.
```

Comment:
Maybe?

```suggestion
        quartile (Q3) of the data, with a line at the median. The
        whiskers extend from the box to 1.5x the inter-quartile 
        range (IQR) distance from the median. Flier points
        are those past the end of the whiskers.
```

Reply:
And actually I'm confused - is the ASCII art also incorrect?

Reply:
No!
It's 1.5x IQR from the edge of the box, so `low = Q1 - 1.5 * IQR` location of the median has no influence here

Reply:
OK, well, its still not clear then, but at least the ASCII art is correct...

Reply:
Also the whole point of this change was to explain why sometimes whiskers are asymmetric (see https://github.com/matplotlib/matplotlib/issues/25134#issue-1568623320) and the proposal above https://github.com/matplotlib/matplotlib/pull/25135#discussion_r1095012398 doesn't resolve that.